receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/constant.go

```diff
@@ -15,13 +15,24 @@
 package awsecscontainermetrics
 
 const (
-	AttributeECSDockerName   = "aws.ecs.docker.name"
-	AttributeECSCluster      = "aws.ecs.cluster.name"
-	AttributeECSTaskARN      = "aws.ecs.task.arn"
-	AttributeECSTaskID       = "aws.ecs.task.id"
-	AttributeECSTaskFamily   = "aws.ecs.task.family"
-	AttributeECSTaskRevesion = "aws.ecs.task.version"
-	AttributeECSServiceName  = "aws.ecs.service.name"
+	AttributeECSDockerName         = "aws.ecs.docker.name"
+	AttributeECSCluster            = "aws.ecs.cluster.name"
+	AttributeECSTaskARN            = "aws.ecs.task.arn"
+	AttributeECSTaskID             = "aws.ecs.task.id"
+	AttributeECSTaskFamily         = "aws.ecs.task.family"
+	AttributeECSTaskRevision       = "aws.ecs.task.version"
+	AttributeECSServiceName        = "aws.ecs.service.name"
+	AttributeECSTaskPullStartedAt  = "aws.ecs.task.pull_started_at"
+	AttributeECSTaskPullStoppedAt  = "aws.ecs.task.pull_stopped_at"
+	AttributeECSTaskKnownStatus    = "aws.ecs.task.known_status"
+	AttributeECSTaskLaunchType     = "aws.ecs.task.launch_type"
+	AttributeContainerImageVersion = "aws.ecs.container.image.version"
```

Comment:
can use `container.image.tag` following the semantic-conventions.

Reply:
Sure